packages/quantic/force-app/main/default/lwc/quanticGeneratedAnswerContent/__tests__/quanticGeneratedAnswerContent.test.js

```diff
@@ -0,0 +1,120 @@
+// @ts-ignore
+import QuanticGeneratedAnswerContent from 'c/quanticGeneratedAnswerContent';
+import {createElement} from 'lwc';
+
+const mockMarkedUse = jest.fn();
+const mockMarkedParse = jest.fn((text) => text);
+global.marked = {
+  use: mockMarkedUse,
+  parse: mockMarkedParse,
+};
+
+const defaultOptions = {
+  isStreaming: false,
+  answerContentFormat: 'text/plain',
+  answer: '',
+};
+
+function createTestComponent(options = defaultOptions) {
+  const element = createElement('c-quantic-generated-answer-content', {
+    is: QuanticGeneratedAnswerContent,
+  });
+  for (const [key, value] of Object.entries(options)) {
+    element[key] = value;
+  }
+
+  document.body.appendChild(element);
+  return element;
+}
+
+// Helper function to wait until the microtask queue is empty.
+function flushPromises() {
+  // eslint-disable-next-line @lwc/lwc/no-async-operation
+  return new Promise((resolve) => setTimeout(resolve, 0));
+}
+
+describe('c-quantic-generated-answer-content', () => {
+  function cleanup() {
+    // The jsdom instance is shared across test cases in a single file so reset the DOM
+    while (document.body.firstChild) {
+      document.body.removeChild(document.body.firstChild);
+    }
+    jest.clearAllMocks();
+  }
+
+  afterEach(() => {
+    cleanup();
+  });
+
+  describe('text/plain answer content', () => {
+    it('should render a simple text/plain answer when not streaming', async () => {
+      const element = createTestComponent({
+        ...defaultOptions,
+        answer: 'Hello, world!',
+      });
+      await flushPromises();
+
+      const answerContent = element.shadowRoot.querySelector(
+        'span.generated-answer-content__answer'
```

Comment:
I suggestion creating a `selectors` object at the top of this file to avoid re-writing these multiple times.